src/map/clif_packetdb.hpp

```diff
@@ -1918,7 +1918,7 @@
 	parseable_packet(0x0974,2,clif_parse_merge_item_cancel,0); // CZ_CANCEL_MERGE_ITEM
```

Comment:
The ones that you converted to struct should have their `packet( packet_id, length )` definition removed, since they work with sizeof now.